sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/Infrastructure/ConversationAnalysisClientWithLanguageSamples.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Was there a reason for this base class? It doesn't seem to add any capabilities that are useful for derivative classes.

Reply:
You can (and probably should) just use the same partial class in different files and add methods. Partial classes can be spread across many, many files.